### PR TITLE
[add] wall components

### PR DESCRIPTION
**Pull request by VahanSargsyan**

### CODE REVIEW

**Review by MyasnikKhachkalyan — APPROVED** [review]

File: src/components/walls/BrickWall/BrickWall.module.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.brick {
+  margin: 0;
+  width: 25px;
+  height: 25px;
```
Comment (hakobpogh) [inline src/components/walls/BrickWall/BrickWall.module.scss]: try using css var `--unit-size`

File: src/components/walls/BrickWall/BrickWall.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React, {FC} from 'react';
+
+import style  from './BrickWall.module.scss';
+import brick from '../../../assets/images/walls/wall_1.svg'
```
Comment (hakobpogh) [inline src/components/walls/BrickWall/BrickWall.tsx:4]: try to understand if the loading images will be delayed in this way or not. Try testing with some throttling and cleared caches.

File: src/components/walls/BrickWall/BrickWall.module.scss (hunk 1 above)
Comment (hakobpogh) [reply src/components/walls/BrickWall/BrickWall.module.scss]: Also, I've suggested some other solution. Please take a look at Myasnik's PR

File: src/components/walls/BrickWall/BrickWall.module.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.brick {
+  margin: 0;
```
Comment (hakobpogh) [inline src/components/walls/BrickWall/BrickWall.module.scss]: why do you need that?

File: src/components/walls/Water/Water.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React, {FC} from 'react';
+
+import style  from './Water.module.scss';
+import water from '../../../assets/images/walls/wall_5.svg'
```
Comment (hakobpogh) [inline src/components/walls/Water/Water.tsx]: Change to waterSvg or waterImg to have more descriptive names

File: src/components/walls/BrickWall/BrickWall.module.scss (hunk 3 above)
Comment (hakobpogh) [reply src/components/walls/BrickWall/BrickWall.module.scss]: @VahanSargsyan I mean why do you need margin here?

**Review by hakobpogh — COMMENTED** [review]
- [ ] Also, some of the file names have spelling issues: bulet -> bullet, sheld -> shield, maybe others too. just take a look at names

File: src/components/walls/BrickWall/BrickWall.module.scss [hunk 5]
```diff
@@ -0,0 +1,4 @@
+.brick {
```
Comment (hakobpogh) [inline src/components/walls/BrickWall/BrickWall.module.scss]: We're using capital letters in CSS too. please. I know this is hard. Consider this for all CSS classes.

**Review by hakobpogh — APPROVED** [review]

**Review by hospogh — APPROVED** [review]

**Pull request merged**